voxelbotutils/cogs/utils/interactions/components/models.py

```diff
@@ -81,7 +82,7 @@ class ComponentHolder(BaseComponent):
     A message component that holds other message components.
     """
 
-    def __init__(self, *components: typing.List[BaseComponent]):
+    def __init__(self, *components: typing.Union[BaseComponent, ActionRow]):
```

Comment:
ActionRow inherits from BaseComponents - this should just be `*components: BaseComponent` so as to futureproof for other layout components.

Reply:
That's true, updated.